interfaces/api/reloc/IKeyframeRetriever.h

```diff
@@ -53,26 +54,33 @@ class XPCF_IGNORE IKeyframeRetriever :
     virtual FrameworkReturnCode addKeyframe(const SRef<SolAR::datastructure::Keyframe> keyframe, bool useMatchedDescriptor=false) = 0;
 
 	/// @brief Suppress a keyframe from the retrieval model
-	/// @param[in] keyframe_id: the keyframe to supress from the retrieval model
+	/// @param[in] keyframeId: the keyframe to supress from the retrieval model
 	/// @return FrameworkReturnCode::_SUCCESS if the keyfram adding succeed, else FrameworkReturnCode::_ERROR_
-    virtual FrameworkReturnCode suppressKeyframe(uint32_t keyframe_id) = 0;
+    virtual FrameworkReturnCode suppressKeyframe(uint32_t keyframeId) = 0;
 
 
     /// @brief Retrieve a set of keyframes close to the frame pass in input.
     /// @param[in] frame: the frame for which we want to retrieve close keyframes.
-    /// @param[out] retKeyframes_id: a set of keyframe ids which are close to the frame pass in input
+    /// @param[out] retKeyframeId: a set of keyframe ids which are close to the frame pass in input
```

Comment:
- Does it make sense to keep this method since we add the one that also returns the scores? If not: either remove or keep for some times by deprecating it with ` [[deprecated]]`, to be removed in a future release. Or do we want to keep it as a convenient method to not have to deal with `std::pair`s when it's not needed...?
- It's not a set, it's a list (vector) in the signature. It's not bad, just since we don't have any notion of score and sort here, it coud indeed be  a set. Unless, we document the fact that this method returns a list of ids sorted by score. Which is a hint at implementing it as a call to the method below. Implementation that could then be given by the Framework since it would just be the same in all implementing components, problably).

Reply:
this method is in fact used in mapping & reloc, I prefer to keep it for the moment. it returns a vector of ids sorted by the corresponding scores. I will add a comment about retKeyframeId in @param[out] retKeyframeId